docs/executor_tutorial/google_lifesciences.rst

```diff
@@ -0,0 +1,897 @@
+We won't need to clone data from `snakemake-tutorial-data <https://github.com/snakemake/snakemake-tutorial-data>`_
+because we will be using publicly accessible data on Google Storage.
+
+First, how does a working directory work for this executor? The present
+working directory, as identified by Snakemake that has the Snakefile, and where
+a more advanced setup might have a folder of environment specifications (env) a folder of scripts 
+(scripts), and rules (rules), is considered within the context of the build.
+When the Google Life Sciences executor is used, it generates a build package of all
+of the files here (within a reasonable size) and uploads those to storage. This
+package includes the .snakemake folder that would have been generated locally.
+The build package is then downloaded and extracted by each cloud executor, which
+is a Google Compute instance.
+
+We next need an `environment.yml` file that will define the dependencies
+that we want installed with conda for our job. If you cloned the "snakemake-tutorial-data"
+repository you will already have this, and you are good to go. If not, save this to `environment.yml`
+in your working directory:
+
+.. code:: yaml
+
+    channels:
+      - conda-forge
+      - bioconda
+    dependencies:
+      - bioconda::snakemake-minimal =5.4.5
+      - python =3.6
+      - jinja2 =2.10
+      - networkx =2.1
+      - matplotlib =2.2.3
+      - graphviz =2.38.0
+      - bcftools =1.9
+      - samtools =1.9
+      - bwa =0.7.17
+      - pysam =0.15.0
+    
+
+Notice that we reference this `environment.yml` file in the Snakefile below.
+Importantly, if you were optimizing a pipeline, you would likely have a folder
+"envs" with more than one environment specification, one for each step.
+This workflow uses the same environment (with many dependencies) instead of
+this strategy to minimize the number of files for you.
+
+The Snakefile then has the following content. It's important to note
+that we have not customized this file from the basic tutorial to hard code 
+any storage. We will be telling snakemake to use the remote bucket as 
+storage instead of the local filesystem.
+
+.. code:: python
+
+    SAMPLES = ["A", "B"]
+
+    rule all:
+        input:
+            "plots/quals.svg"
+
+    rule bwa_map:
+        input:
+        fastq="samples/{sample}.fastq",
+        idx=multiext("genome.fa", ".amb", ".ann", ".bwt", ".pac", ".sa")
+    conda:
+        "environment.yml"
+    output:
+        "mapped_reads/{sample}.bam"
+    params:
+        idx=lambda w, input: os.path.splitext(input.idx[0])[0]
+    shell:
+        "bwa mem {params.idx} {input.fastq} | samtools view -Sb - > {output}"
+
+    rule samtools_sort:
+        input:
+            "mapped_reads/{sample}.bam"
+        output:
+            "sorted_reads/{sample}.bam"
+        conda:
+            "environment.yml"
+        shell:
+            "samtools sort -T sorted_reads/{wildcards.sample} "
```

Comment:
How can this work actually? Don't we run into the issue with --default-remote-prefix introducing an additional prefix, that is not reflected here in the arg passed to `-T`?

Reply:
Never mind, now I understand again. The -T is for the temp files, hence it is actually fine.